src/test/java/org/apache/drill_web_test_framework/web_ui/pages/OptionsPage.java

```diff
@@ -0,0 +1,244 @@
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class OptionsPage extends BasePage {
+
+  @FindBy(css = "input[id='searchBox']")
+  private WebElement searchField;
+
+  @FindBy(css = "button[title='Clear search']")
+  private WebElement clearSearchButton;
+
+  @FindBy(id = "optionsTbl")
+  private WebElement optionTable;
+
+  @FindBy(xpath = "//button[text()='planner']")
+  private WebElement plannerFilter;
+
+  @FindBy(xpath = "//button[text()='store']")
+  private WebElement storeFilter;
+
+  @FindBy(xpath = "//button[text()='parquet']")
+  private WebElement parquetFilter;
+
+  @FindBy(xpath = "//button[text()='hashagg']")
+  private WebElement hashaggFilter;
+
+  @FindBy(xpath = "//button[text()='hashjoin']")
+  private WebElement hashjoinFilter;
+
+  @FindBy(xpath = "//th[text()='OPTION']")
+  private WebElement optionSortBy;
+
+  @FindBy(xpath = "//th[text()='VALUE']")
+  private WebElement valueSortBy;
+
+  @FindBy(xpath = "//th[text()='DESCRIPTION']")
+  private WebElement descriptionSortBy;
+
+  @FindBy(css = "button[onclick=\"alterSysOptionUsingId('drill.exec.functions.cast_empty_string_to_null')\"]")
+  private WebElement updateBooleanButton;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select")
+  private WebElement booleanDropdown;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select>option[value=\"true\"]")
+  private WebElement booleanDropdownTrue;
+
+  @FindBy(css = "button[onclick=\"alterSysOption('drill.exec.functions.cast_empty_string_to_null','false', 'BOOLEAN')\"]")
+  private WebElement booleanDefaultButton;
+
+  @FindBy(css = "form[id='drill.exec.memory.operator.output_batch_size']>div>div>input")
+  private WebElement numericField;
+
+  @FindBy(css = "button[onclick=\"alterSysOptionUsingId('drill.exec.memory.operator.output_batch_size')\"]")
+  private WebElement numericUpdateButton;
+
+  @FindBy(css = "button[onclick=\"alterSysOption('drill.exec.memory.operator.output_batch_size','16777216', 'LONG')\"]")
+  private WebElement numericDefaultButton;
+
+  @FindBy(css = "form[id='drill.exec.storage.file.partition.column.label']>div>div>input")
+  private WebElement stringField;
+
+  @FindBy(css = "button[onclick=\"alterSysOptionUsingId('drill.exec.storage.file.partition.column.label')\"]")
+  private WebElement stringUpdateButton;
+
+  @FindBy(css = "button[onclick=\"alterSysOption('drill.exec.storage.file.partition.column.label','dir', 'STRING')\"]")
+  private WebElement stringDefaultButton;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select>option[value='true']:checked")
+  private WebElement valueOfBooleanTrue;
+
+  @FindBy(css = "form[id='drill.exec.functions.cast_empty_string_to_null']>div>div>select>option[value='false']:checked")
+  private WebElement valueOfBooleanFalse;
+
+  @FindBy(css = "input[value='16777217']")
```

Comment:
Default value can change. Better search by the option name.

Reply:
Done